src/components/FormSteps/FormSteps.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { arrayRange } from 'utils/arrayRange';
+
+interface Props {
+  finalStep: number;
+  currentStep: number;
+}
+
+const FormSteps: FC<Props> = ({ finalStep, currentStep }) => {
+  const [step, setStep] = useState(1);
+
+  const stepArray = arrayRange(1, finalStep, 1);
+
+  useEffect(() => {
+    setStep((prev) => (currentStep > prev ? currentStep : prev));
+  }, [currentStep]);
+
+  return (
+    <div className='w-full flex justify-center mb-10'>
+      {stepArray.map((value) => (
```

Comment:
For you information. You can use not only _value_ in param of map. Will be better if you add more informative variable names in future. Not required to change this now

Reply:
Also in setState above. Better to name more informative. (like previousStep maybe)